getlyrics/getlyrics.py

```diff
@@ -23,6 +23,14 @@ def prompt(*args):
         sys.stdout = old_stdout
 
 
+def pager(string):
+    import subprocess
+    pager_prog = os.environ['PAGER']
```

Comment:
Can you change this to `pager_prog = os.environ['PAGER'] if 'PAGER' in os.environ else 'less'`?